### ISSUE TITLE
1.4.2 bower.json 

### ISSUE THREAD

**Issue by mromanuk** [harvesthq/chosen#2324]
The version in `bower.json` is incorrect, please also review the main section

`chosen.css` should be `chosen.min.css` 
```json
  "main": [
    "chosen.jquery.min.js",
    "chosen.css",
    "chosen-sprite@2x.png",
    "chosen-sprite.png"
  ],
```
These are the files included with a `chosen.min.css`
```
total 104
-rw-r--r--  1 martin  staff   1164 Mar 28 01:02 bower.json
-rw-r--r--  1 martin  staff    538 Mar 28 01:02 chosen-sprite.png
-rw-r--r--  1 martin  staff    738 Mar 28 01:02 chosen-sprite@2x.png
-rw-r--r--  1 martin  staff  27969 Mar 28 01:02 chosen.jquery.min.js
-rw-r--r--  1 martin  staff  11115 Mar 28 01:02 chosen.min.css
```

**Comment by wefarrell** [comment]
Seeing this issue as well.

**Comment by stof** [comment]
the 1.4.2 tag is indeed messed up in the bower-chosen repo.

@pfiller looks like the wrong commit was tagged. you may want to check the release script

**Comment by Bishbulb** [comment]
This is causing us problems as well. Can a hotfix be published with just this fix?

**Comment by dannykopping** [comment]
Which commit should it tagged against?

**Comment by lambert-velir** [comment]
+1

**Comment by raywang945** [comment]
Causing problems as well

**Comment by robertjlooby** [comment]
^^ still messed up

**Comment by tjschuck** [comment]
@pfiller FWIW, I think we should just release a 1.4.3 version that is fixed, and not try to mutate 1.4.2.  Does bower have any ability to "yank" bad versions?

**Comment by thisischrisj** [comment]
+1

**Comment by marcelaerts** [comment]
+1

**Comment by vjanssens** [comment]
Quick fix for 1.4.2. Add the 'overrides' object to your bower.json file to override Chosen's main object:

````json
{
  "dependencies": {
    "jquery": "~2.1.1",
    "chosen": "~1.4.2"
  },
  "overrides": {
    "chosen": {
      "main": [
        "chosen.jquery.min.js",
        "chosen.min.css"
      ]
    }
  }
}
````

**Comment by JonCatmull** [comment]
Same issue please fix

**Comment by sylvainbaronnet** [comment]
+1

**Comment by a-k-g** [comment]
+1

**Comment by ashokfernandez** [comment]
+1 please push a new version to fix this

**Comment by koenpunt** [comment]
I believe this issue has been resolved in 00f2e25d3a0d0ba3a0b21494d635c04bcfc55338

**Comment by tjschuck** [comment]
Well, that + pushing a new version :)  But I think that's coming soon...

I'm going to close this, since the next version that goes out will fix it.  Thanks for the feedback, everyone!
